scripts/build.py

```diff
@@ -174,47 +184,113 @@ def _ensure_files_exist(filepaths):
         filepaths: list(str). Paths to files that we want to ensure exist.
 
     Raises:
-        Exception: Raised if one or more of the files does not exist.
+        OSError: One or more of the files does not exist.
     """
     for filepath in filepaths:
         if not os.path.isfile(filepath):
-            raise Exception('File %s does not exist.' % filepath)
+            raise OSError('File %s does not exist.' % filepath)
+
+
+def get_file_count(directory_path):
+    """Count total number of file directory, subtracting ignored files.
+
+    Args:
+        directory_path: str. Directory to be walked.
 
+    Returns:
+        int. Total number of files minus ignored files.
+    """
+    total_file_count = 0
+    for _, _, files in os.walk(directory_path):
```

Comment:
So this os.walk naming convention was used in the `os` Python doc's example, 
https://docs.python.org/2/library/os.html#os.walk
```Python
for root, dirs, files in os.walk('python/Lib/email'):
...
```
Should it be
```Python
for dirpath, dirnames, filenames in os.walk('python/Lib/email'):
```
instead?

Reply:
I think dirpath or root is fine for the first one.

For the second one, I'm not sure. Is it just the "final part" of the dirpath? If so then dirnames sounds good. Otherwise maybe dirpaths.

Third one should definitely be filenames.

Reply:
Yep. Re-named all variables used with os.walk.